### ISSUE TITLE
Scripts for peram_gen

### ISSUE THREAD

**Issue by maowerner** [HISKP-LQCD/SoBERH#8]
Could we upload the scripts for peram_gen from Bartek (slurm_create_runs.sh, ... and templates) into 
this repo? I would like to have a central place to look for scripts someone might have already written.

**Comment by kostrzewa** [comment]
Yes, I agree. It will take some organisation, though. There's no "one size fits all" approach.
